AddNewLHComponent element made with example bundle files and go file

jarpinosynthace opened the pull request.

chr1sgrant commented:
@synthacebot test this please

chr1sgrant commented on an/Utility/AddNewLHComponent.an:
```diff
@@ -0,0 +1,103 @@
+	var err error
+
+	//Store the TemplateComponent name into a variable for text output at the end of the run
+	TemplateComponentName := TemplateComponent.CName
```
Should this be exported?

jarpinosynthace commented:
@chr1sgrant I have changed the code slightly, i had made a mistake. As it was it would have exported the TemplateLHComponent name which we dont really need. We need to export the NewLHComponent name. The export comment i had meant to say that it was saving the template name as a variabkle to be used in the Status output. I have updated the code so this is now not capitalised and that we now actually output in Data the NewLHComponentName

chr1sgrant commented:
@synthacebot test this please

chr1sgrant commented:
@synthacebot test this please
